### PR TITLE
[INTEL MKL] DNNL 1.0 op support for Softmax, Identity_op, and Lrn ops.

### PR DESCRIPTION
**Pull request by LakshayT**
Added changes to the softmax, identity_op and Lrn op kernels for DNNL 1.0 support.

### CODE REVIEW

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! I have one minor nit and another comment about the refactoring that might happen in #36496. If you don't want to wait for that PR, I'm also fine with adding TODOs in this PR to call the refactored method later (if we end up having one). Let me know.

File: tensorflow/core/kernels/mkl_softmax_op.cc [hunk 1]
```diff
@@ -61,9 +63,18 @@ class MklSoftmaxPrimitive : public MklPrimitive {
...
+#ifdef ENABLE_MKLDNN_V1
+    DCHECK_EQ(context_.fwd_primitives.size(),
+              context_.fwd_net_args.size());
+    for (size_t i = 0; i < context_.fwd_primitives.size(); ++i) {
+      context_.fwd_primitives.at(i).execute(*context_.fwd_stream,
+                                            context_.fwd_net_args.at(i));
+    }
+#else
     context_.fwd_stream->submit(context_.fwd_primitives);
+#endif
```
[the remark this answers is not in the document]
Comment (penpornk) [reply tensorflow/core/kernels/mkl_softmax_op.cc]: PR #36496 is merged. Please sync with master and call the refactored method (`execute_primitives`).

[the remark this answers is not in the document]
Comment (LakshayT) [reply tensorflow/core/kernels/mkl_softmax_op.cc]: I have pushed the new changes. Could you please review them?

[the remark this answers is not in the document]
Comment (penpornk) [reply tensorflow/core/kernels/mkl_softmax_op.cc]: The changes aren't reflected here yet. Could you please push again? You can go to the `Commits` or the `Files changed` tabs to check.

**Review by penpornk — APPROVED** [review]
I see them now. Thank you for the changes!
